binding/default_validator.go

```diff
@@ -18,10 +18,10 @@ type defaultValidator struct {
 	validate *validator.Validate
 }
 
-type sliceValidateError []error
+type SliceValidateError []error
```

Comment:
How about renaming it to a more grammatically correct form at the same time:
```suggestion
type SliceValidationError []error
```

Reply:
Yep, done.